### DISCUSSION TITLE
Reversing for OS-level Scroll-to-Zoom broken since OSX 12.6?

### DISCUSSION THREAD

**Discussion by baritchi** [pilotmoon/Scroll-Reverser#155] [Q&A]
Hey y'all, wondering if this is a pervasive issue or just me. I've been using Scroll Reverser for a little over a year now on my Mac in clamshell mode with an external mouse, no problems. I also use the OS-level Scroll to Zoom from Accessibility options (see screenshot), where you hold down a hotkey (for me, the command key) and Scroll your mouse or trackpad to zoom in on the screen. Up until sometime this week - I think since the 12.6 update - I scrolled my wheel up to zoom in, and down to zoom out.  

As of now, Scroll Reverser continues to work correctly in every *other* use case except OS level zoom scrolling... I can turn it off and the scroll wheel feels "backwards."  I turn it back on, it feels natural. But, turning Scroll Reverser on or off no longer has any bearing on OS Scroll to Zoom direction.  When I hold down the Command key, it now zooms out when I scroll the mouse wheel up, and zooms in when I scroll the mouse wheel down.. which of course feels "backwards." And no matter what I do with Scroll Reverser, it is always in this direction.  

So, I'm guessing Apple changed something in 12.6 in how it handles the Scroll wheel input and it's skipping Scroll Reverser now when you hold down the Scroll to Zoom hotkey. 

Thoughts?  Ideas?

Thanks! Cheers,
Andi 

![Screen Shot 2022-09-30 at 11 45 53 AM](https://user-images.githubusercontent.com/67607826/193318689-1f57543f-b81f-4bad-a936-e19a72c1e2f1.png)

**Comment by pilotmoon** [comment]
Honestly I have no idea, but you theory sounds quite likely that they changed the point at which the scrolling events are processed. I'm not able to spend much time on Scroll Reverser at the moment I'm afraid, but when I do I will look into this.

**Reply by pilotmoon** [reply]
I've added this as an issue so I can find it again.
